third-party/realdds/include/realdds/topics/dds-topic-names.h

```diff
@@ -26,6 +26,29 @@ constexpr char const * METADATA_TOPIC_NAME = "/metadata";
 constexpr char const * DFU_TOPIC_NAME = "/dfu";
 
 
+namespace ros2 {
+
+constexpr char const * NAMESPACE = "/realsense";  // must begin with /
+
+constexpr char const * ROOT = "rt/";
```

Comment:
`Root` is too generic and confusing with what you ended up calling `namespace`. Maybe call `ROS_ROOT` or `ROS_TOPIC_ROOT`